reference/private-networking.html.md

```diff
@@ -92,7 +92,7 @@ Examples of retrieving this information are in the [fly-examples/privatenet](htt
 
 ## Private Network VPN
 
-Along with our [6PN private networking](/docs/reference/private-networking/),  [WireGuard](https://wireguard.com/) networking software. This is a flexible and secure way to plug into each one of your Fly organizations and connect to any and all apps within that organization.
+In conjunctin with our [6PN private networking](/docs/reference/private-networking/), we can use [WireGuard](https://wireguard.com/) VPN software. This is a flexible and secure way to plug into each one of your Fly organizations and connect to any and all apps within that organization.
```

Comment:
There's a typo in "conjunction"--but would it be easier if we just say "You can use [WireGuard](https://wireguard.com/) VPN software to connect to our [6PN private network](https://fly.io/docs/reference/private-networking/) from outside."?

Reply:
ooph and also yes, agree!